File: c++/operators/operators_increment_and_decrement.cpp
#include <iostream>

int main()
{
  int a = 6, b = 3;

  std::cout << "a = " << a << ", b = " << b << std::endl;
  // preincrement
  std::cout << "++a : " << ++a << "(" << a << ")" << std::endl;
  // predecrement
  std::cout << "--b : " << --b << "(" << b << ")" << std::endl;

  std::cout << "a = " << a << ", b = " << b << std::endl;
  // postincrement
  std::cout << "a++ : " << a++ << "(" << a << ")" << std::endl;
  // postdecrement
  std::cout << "b-- : " << b-- << "(" << b << ")" << std::endl;
}

File: javascript/operators/operators_increment_and_decrement.js
#!/usr/bin/env node

let a = 6, b = 3;

console.log(`a = ${a}, b = ${b}`);
// preincrement
console.log(`++a : ${++a}(${a})`);
// predecrement
console.log(`--b : ${--b}(${b})`);

console.log(`a = ${a}, b = ${b}`);
// postincrement
console.log(`a++ : ${a++}(${a})`);
// postdecrement
console.log(`b-- : ${b--}(${b})`);
